tasks/install.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create temp directory for doing work in
+  tempfile:
+    state: directory
+  register: tempdir
+  changed_when: false
+
+- block:
+    - name: Copy templates
+      with_items:
```

Comment:
`with_list` is better unless one wants to merge contained arrays.

Reply:
Interestingly enough the Ansible Docs for 2.4 have no mentions of either `with_list` nor a `list` lookup plugin so I thought it does not exist, however the docs for 2.5 describe how to migrate from `with_list` to `loop`.